nodeup/pkg/model/kube_apiserver.go

```diff
@@ -298,6 +298,9 @@ func (b *KubeAPIServerBuilder) buildPod() (*v1.Pod, error) {
 	}
 
 	kubeAPIServer.ClientCAFile = filepath.Join(b.PathSrvKubernetes(), "ca.crt")
+	if b.Cluster.Spec.KubeAPIServer.ClientCAFile != "" {
```

Comment:
We have `kubeAPIServer := b.Cluster.Spec.KubeAPIServer` above, and then you set `		kubeAPIServer.ClientCAFile = b.Cluster.Spec.KubeAPIServer.ClientCAFile`

Is this rather redundant or am I missing something?

Reply:
Thanks for your comment @olemarkus.

Before the change, line 300 would overwrite `kubeAPIServer.ClientCAFile` to `filepath.Join(b.PathSrvKubernetes(), "ca.crt")` regardless.  

I did not realize we have kubeAPIServer := b.Cluster.Spec.KubeAPIServer above. A better change might be overwrite the  `kubeAPIServer.ClientCAFile` to `filepath.Join(b.PathSrvKubernetes(), "ca.crt")` only when `b.Cluster.Spec.KubeAPIServer.ClientCAFile` is empty string (default)?